docs/source/integrations/pyspark.rst

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
Do we need a table of contents here? In other words, could `integrations/pyspark_integration.md` become this page, hence having only one?

(I suppose if we want to grow this in the future, it might be a good idea to keep it just to avoid creating one redirect, but wanted to ask anyway)

Reply:
We probably don't need it and could instead just re-title `pyspark_integration.md` to have a header that corresponds with the one for databricks (That is retitle to "Pyspark integration")